zingo-testutils/src/lib.rs

```diff
@@ -737,6 +738,131 @@ pub mod scenarios {
         )
     }
 
+    pub async fn funded_orchard_sapling_transparent_shielded_mobileclient(
+        value: u64,
```

Comment:
i think it would be more useful to have a separate scenario called `funded_mobileclient` with three inputs, `orchard_funds`, `sapling_funds` and `transparent funds`. this can replace `funded_orchard_mobileclient` and also be nested into this scenario. I am happy to approve this PR without this work done now but maybe i the future we can refactor?

Reply:
oh, sounds really smart, but I don't know if my rust skills are good enough to do that. But I'm happy to do this in some near future... as I said, I'm planning to create a few more tests & I think I'll do this with your help for those new tests... How sounds that?

Reply:
great! yeah lets work together on this upgrade after this PR